mtags-shared/src/main/scala/scala/meta/internal/pc/InlayHints.scala

```diff
@@ -123,6 +181,36 @@ object InlayHints {
   }
 }
 
+final case class InlayHintBlock(
+    indentLevel: Int,
+    hints: List[BlockInlayHint]
+) {
+  def build: List[BlockInlayHint] = {
+    if (hints.length == 1) {
```

Comment:
I tried to do this filtering in the `unapply` but I would need the LSP positions and when I tried to access them at that point the tests all failed with a scary "compiler broken!! :c" message so I decided to just do it here